.github/ISSUE_TEMPLATE/site_showcase.md

```diff
@@ -0,0 +1,96 @@
+---
+name: Site Showcase Submission
+about: Want to submit a site to the site showcase? This is the place to submit it.
+---
+
+<!--
+  To make sure your site gets submitted to the showcase, please follow the suggested format below.
+
+  Before submitting a site to the site showcase, please make sure no on else has already submitted it by searching existing PRs: https://github.com/gatsbyjs/gatsby/pulls
+-->
+
+## Steps
+
+1. Create a branch in master called site-showcase-submission
```

Comment:
Perhaps we should just encourage people to edit the yaml file on GitHub? That'd probably be easier.

Reply:
👍 Chatted about this with @pieh yesterday and came to that conclusion, too:

* link the "Submit your site" button to a (this) markdown file with instructions
* the description contains a link to edit `sites.yml` directly on GitHub